ui/src/components/CloneDialog.tsx

```diff
@@ -87,7 +87,7 @@ export default function CloneDialog({ ...props }) {
   };
 
   return (
-    <Dialog open={props.open} onClose={props.onClose}>
+    <Dialog fullWidth maxWidth="sm" open={props.open} onClose={props.onClose}>
```

Comment:
I would get rid of the text line 102 as it is redundant with the title

Reply:
I am waiting for design input on this dialog, before making any change.